lib/rspec/core/world.rb

```diff
@@ -21,6 +21,16 @@ def initialize(configuration=RSpec.configuration)
         configuration.world = self
         @example_groups = []
         @example_group_counts_by_spec_file = Hash.new(0)
+        filter_examples
+      end
+
+      # @api private
+      #
+      # Filter examples.
+      #
+      # This is a separate method so that filters can be modified/replaced and
+      # examples refiltered during a process's lifetime.
+      def filter_examples
```

Comment:
I think `prepare_example_filtering` is a better name for this, given when this method returns, no filtering has actually been performed (as it happens lazily).

Reply:
Sure thing.